### PR TITLE
[6.4.x] BZ-1310439 - RuntimeManagerFactory may fail to initialize in OSGi

### PR DESCRIPTION
**Pull request by mswiderski**
(cherry picked from commit 9e76430b72904b8b1e1e4c2b9d73ffc955314519)

### CODE REVIEW

File: jbpm-flow/src/main/java/org/jbpm/osgi/flow/core/Activator.java [hunk 1]
```diff
@@ -42,13 +43,14 @@ public void start(BundleContext bc) throws Exception {
         this.processRuntimeReg2 = bc.registerService( new String[]{ ProcessMarshallerFactoryService.class.getName(), Service.class.getName()},
                                                                    new ProcessMarshallerFactoryServiceImpl(),
                                                                    new Hashtable() );
-        ProcessRuntimeFactory.resetInitialization();
+        RuntimeManagerFactory.Factory.reset();
+        ProcessBuilderFactory.resetInitialization();
```
Comment (psiroky) [inline jbpm-flow/src/main/java/org/jbpm/osgi/flow/core/Activator.java]: Is this change intentional - `ProcessRuntimeFactory -> ProcessBuilderFactory`? It seems to be only here, not in the master PR.

Comment (mswiderski) [inline jbpm-flow/src/main/java/org/jbpm/osgi/flow/core/Activator.java]: ok, my bad merge, fixing it now....

**Comment by mswiderski** [comment]
jenkins reset this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
merged into 6.4.x
